packages/protovalidate/package.json

```diff
@@ -42,7 +42,7 @@
     }
   },
   "dependencies": {
-    "@bufbuild/cel": "^0.1.0"
+    "@bufbuild/cel": "0.2.0"
```

Comment:
We're pinning to an exact version because we do not want breaking changes to affect users of protovalidate.